pkg/apis/starrocks/v1/load_type.go

```diff
@@ -11,6 +11,9 @@ type loadInterface interface {
 	GetNodeSelector() map[string]string
 	GetAffinity() *corev1.Affinity
 	GetTolerations() []corev1.Toleration
+	GetStartupProbe() *corev1.Probe
```

Comment:
I would rather allow the user to specify the total tolerant of times for the probe, instead of specifying a different probe, which is only valuable for the users with deep understanding of sr deployment. Let's simplify it.